### DISCUSSION TITLE
Personalization

### DISCUSSION THREAD

**Discussion by TAPD2023** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#845] [Q&A]
Hello,
I'm german and I beg pardon for my english.
I tested the desktop app for WIN10 and generally like it very much. But I am not able to use personalized questions and answers with the assistant. Everytime I try, my smartphone opens to allow personal answers but since the app (the notebook) has no devicename, it is not possible to grant rights to the app. Can anyone help?
Thanks
TAPD

**Comment by im-coder-lg** [comment]
Hey up TAPD,

As of now, personal results seem to be a bit on-and-off. GAUDC is unofficial and our "loophole" for activating Personal Results seem to be blocked, i. e. Results are accepted on devices but is not seen on the list. So it seems like it is a problem right now. We are trying to find a fix to it, as personal results was one of the biggest things here.

Hope it helps!

**Comment by sumeshir26** [comment]
@TAPD2023 Try this: https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues/784#issuecomment-1370355169

**Reply by im-coder-lg** [reply]
Okay, I gotta do some work and add this too.

**Reply by sumeshir26** [reply]
@im-coder-lg I haven't actually tested this, Cuz I don't have a phone with the same google account as the PC, so can you test this if possible?

**Reply by im-coder-lg** [reply]
Yep, I will do that on Saturday. If it works, I swear I will make an up-to-date public issues list - but I need help, so you got ideas on public issues here?

**Reply by sumeshir26** [reply]
> Yep, I will do that on Saturday. If it works, I swear I will make an up-to-date public issues list - but I need help, so you got ideas on public issues here?

This and the `sessionStorage` thing

**Reply by im-coder-lg** [reply]
Any issue links?
